eng/testing/netfx.exe.config

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <runtime>
-    <developmentMode developerInstallation="true" />
```

Comment:
Interesting, so we no longer rely on DEVPath and we get all the right bindingRedirects?  That's significant improvement to test coverage.

Reply:
Exactly. There was an issue with them being overwritten but I fixed that.